src/items/generics.md

```diff
@@ -227,6 +227,22 @@ fn generic<const B: bool>() {
 }
 ```
 
+r[items.generics.const.inferred]
+
+r[items.generics.const.inferred.syntax]
+```grammar,const
+InferredConst ->
+      `_`
+    | `(` InferredConst `)`
+```
```

Comment:
I'm not sure I fully understand this grammar since it isn't referred to anywhere.

Am I understanding correctly that this is added for GenericArgsConst? Or is it somewhere else? That is, should it be:

```
GenericArgsConst ->
      BlockExpression
    | LiteralExpression
    | `-` LiteralExpression
    | SimplePathSegment
    | InferredConst

InferredConst ->
      `_`
    | `(` InferredConst `)`
```